Neos.Neos/Classes/Domain/Service/NodeDuplicationService.php

```diff
@@ -179,6 +201,36 @@ public function calculateCopyNodesRecursively(
         return $commands;
     }
 
+    private function resolveUriPathSegmentForCopy(
+        string $originalUriPathSegment,
+        NodeAggregateId $targetParentNodeAggregateId,
+        ContentSubgraphInterface $subgraph
+    ): string {
+        $alreadyClaimedUriPathSegments = [
+            $originalUriPathSegment
+        ];
```

Comment:
this part is weird  - below, $alreadyClaimedUriPathSegments is a key-value map where the KEYS matter; and above the VALUES matter. IMHO this code can just be removed? :)